library/Rules/Json.php

```diff
@@ -15,11 +15,12 @@ class Json extends AbstractRule
 {
     public function validate($input)
     {
-        if (is_string($input)
-            && strtolower($input) == 'null') {
-            return true;
+        if (!is_string($input)) {
+            return false;
         }
 
-        return (null !== json_decode($input));
+        $decode = json_decode($input);
+
+        return is_object($decode) || is_array($decode);
```

Comment:
It that's in the [RFC 7159](https://tools.ietf.org/html/rfc7159) that's what we should do, so let's stick with:

```php
json_decode($input);

return (json_last_error() === JSON_ERROR_NONE);
```



If PHP is able to parse it, that means it's a valid JSON. Can you update this pull request with this change?

PS.: I'm going to read it any way, but I think we should trust PHP for now.